usaspending_api/etl/tests/test_es_rapidloader.py

```diff
@@ -104,7 +119,7 @@ def award_data_fixture(db):
 }
 
 
-def test_es_award_loader_class(db, award_data_fixture, elasticsearch_award_index):
```

Comment:
why did the first parameter go away?

Reply:
Looks like the fixture already includes `db` so these are superfluous.